infra/identity/prod/westeurope/main.tf

```diff
@@ -19,6 +19,56 @@ provider "azurerm" {
   }
 }
 
-data "azurerm_subscription" "current" {}
+resource "azurerm_resource_group" "rg_identity" {
+  name     = "${local.project}-identity-rg"
+  location = local.location
+
+  tags = local.tags
+}
+
+module "federated_identities" {
+  source = "../../../modules/azure_identity_federation_github"
+
+  prefix       = "dx"
+  env_short    = "p"
+  env          = "prod"
+  repositories = ["dx-typescript"]
+  domain       = "typescript"
+
+  continuos_integration = {
+    enable = true
+    roles = {
```

Comment:
can we have this value with a default (within the module) in order to not have to specify these values here?

Reply:
Done